src/background.js

```diff
@@ -1416,7 +1417,8 @@ function initializePopup() {
  */
 function addCommonGhosteryAndAntitrackingListeners() {
 	let urlFilters = ['http://*/*', 'https://*/*', 'ws://*/*', 'wss://*/*'];
-	if (IS_EDGE) {
+	if (IS_EDGE || IS_FIREFOX) {
```

Comment:
@IAmThePan looks like we are going to remove `ws` and `wss` altogether for now.  See cliqz pipeline [here](https://github.com/cliqz/navigation-extension/commit/a5e65dc5d478e542c3aa886216b8ffd0dd690440)